pkgs/development/python-modules/molecule/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools-scm
+, ansible-lint
+, ansible-compat
+, cerberus
+, click
+, click-help-colors
+, click-completion
+, cookiecutter
+, pluggy
+, pyyaml
+, pytestCheckHook
+, selinux-python
+, subprocess-tee
+}:
+
+buildPythonPackage rec {
+  pname = "molecule";
+  version = "3.5.2";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1jp0djj69w0vw2697yr471zdkvd4smggcfvan5z8v6f0wncz0an8";
+  };
+
+  postPatch = ''
```

Comment:
Maybe it is more useful to use [pythonRelaxDeps](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/interpreters/python/hooks/python-relax-deps-hook.sh), as an example it's used at [ansible-doctor](https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/admin/ansible/doctor.nix).